nmostesting/suites/IS0502Test.py

```diff
@@ -724,36 +724,59 @@ def test_14(self, test):
                                             return test.FAIL("Video Flow {} components do not match the expected "
                                                              "dimensions for Sender sampling {}"
                                                              .format(flow["id"], sampling))
-                            elif param_components[0] == "width":
+                                        if index == 0 and component["name"] not in ["Y", "I", "R"]:
```

Comment:
Hmm. Rather than these additions, could we just change above:
```python
if sampling == "4:4:4" or (sampling is not None and index == 0):
```
to something like:
```python
 if sampling == "4:4:4" or sampling is None or component["name"] in ["Y", "I"]:
```
(I'm not sure the handling of `sampling is None` is right at the moment? E.g. 'RGB' should check width/height of every component, right?)

Reply:
Yes, there's definitely an error in my logic there. I'll correct as you suggest.